VMEncryption/main/DiskUtil.py

```diff
@@ -1203,6 +1291,18 @@ def get_azure_devices(self):
                 blk_items.append(current_blk_item)
 
         return blk_items
+    
+    def get_azure_nvme_os_devices(self):
+        devices = []
+        is_os_nvme, os_block_device = self.is_os_disk_nvme()
+        if is_os_nvme:
+            proc_comm = ProcessCommunicator()
+            self.command_executor.ExecuteInBash('ls {0}*'.format(os_block_device), True, communicator=proc_comm)
+            os_devices = re.split('[ \n]', proc_comm.stdout)
+            for os_device in os_devices:
+                if os_device.startswith(os_block_device) and os.path.exists(os_device):
```

Comment:
Is there a case when os_device can be null? Should there be a check to ensure os_device is not null, before invoking these other methods on os_device?

Reply:
If OS is determined to be NVMe then we will always have an OS device.
Also, I am planning to do a review of error handling for this new path. I will add more checks then.